### PR TITLE
Review Storage for the Web

### PR DESCRIPTION
**Pull request by jpmedley**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

-
-
-

When you're ready to submit your PR, don't forget to add the `$-presubmit` label.

### CODE REVIEW

**Comment by jpmedley** [comment]
Friendly ping.

File: src/site/content/en/blog/storage-for-the-web/index.md [hunk 1]
```diff
@@ -150,10 +149,9 @@ if (navigator.storage && navigator.storage.estimate) {
 }
 ```
 
-The StorageManager isn't [implemented][caniuse-sm] in all browsers yet, so you
-must feature detect it before using it. Even when it is available, you must
-still catch over-quota errors (see below). In some cases, it's possible for
-the available quota to exceed the actual amount of storage available.
```
Comment (petele) [inline src/site/content/en/blog/storage-for-the-web/index.md:156]: Do not delete this section, it's still contains critical information.

File: src/site/content/en/blog/storage-for-the-web/index.md [hunk 2]
```diff
@@ -65,9 +66,7 @@ They are synchronous, and are not accessible from web workers. Like
 LocalStorage and SessionStorage, cookies are limited to only strings.
 
 The [File System API][mdn-fileapi] and FileWriter API provide methods for
-reading and writing files to a sandboxed file system. While it is asynchronous,
-it is not recommended because it is
-[only available in Chromium-based browsers][caniuse-fs].
```
Comment (petele) [inline src/site/content/en/blog/storage-for-the-web/index.md:70]: Please keep this.

**Review by petele — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/storage-for-the-web/index.md (hunk 2 above)
Comment (jpmedley) [reply src/site/content/en/blog/storage-for-the-web/index.md:70]: The information for this on caniuse is out of date. Here's [MDN's data](https://developer.mozilla.org/en-US/docs/Web/API/File_and_Directory_Entries_API/Introduction#browser_compatibility).

File: src/site/content/en/blog/storage-for-the-web/index.md (hunk 1 above)
Comment (petele) [reply src/site/content/en/blog/storage-for-the-web/index.md:156]: StorageManager may be, but the required API `.estimate()` is not https://caniuse.com/mdn-api_storagemanager_estimate

File: src/site/content/en/blog/storage-for-the-web/index.md (hunk 2 above)
Comment (petele) [reply src/site/content/en/blog/storage-for-the-web/index.md:70]: I don't trust that and want to verify this myself before accepting that. I'll try to give it a shot tomorrow.

Comment (jpmedley) [reply src/site/content/en/blog/storage-for-the-web/index.md:70]: Let me know when you have an answer.

File: src/site/content/en/blog/storage-for-the-web/index.md (hunk 1 above)
Comment (jpmedley) [reply src/site/content/en/blog/storage-for-the-web/index.md:156]: It still could use some updating. Please see the rewrite immediately below this comment (line 151).

**Comment by jpmedley** [comment]
Ping.

**Comment by rachelandrew** [comment]
Closing as this file has been updated since, if changes are still needed please make a CL on the migrated site.

**Pull request closed without merging**